dashboard/src/app/projects/create-project/add-ssh-key/add-ssh-key.controller.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+/**
+ * This class is handling the controller for the add secret key
+ * @author Oleksii Orel
+ */
+export class AddSecretKeyCtrl {
+
+  /**
+   * Default constructor.
+   * @ngInject for Dependency injection
+   */
+  constructor($mdDialog, cheAPI, lodash) {
+    this.$mdDialog = $mdDialog;
+    this.cheAPI = cheAPI;
+    this.lodash = lodash;
+
+    this.configURL = '#/workspaces';
+
+    // fetch workspaces when initializing
+    let promise = cheAPI.getWorkspace().fetchWorkspaces();
+    promise.then(() => {
+      this.updateConfigURL();
+    }, (error) => {
+      if (error.status === 304) {
+        this.updateConfigURL();
+      }
+    });
+
+  }
+
+  /**
+   * Update config url.
+   */
+  updateConfigURL() {
+    let workspaces = this.cheAPI.getWorkspace().getWorkspaces();
+
+    let ctrl = this;
+    let findWorkspace = this.lodash.find(workspaces, function (workspace) {
```

Comment:
with arrow function, no need to do 'let ctrl = this" we can use inside function return workspace.id === this.workspaceId